tgui-next/packages/common/logging.js

```diff
@@ -1,8 +1,3 @@
-/**
- * Copyright (c) 2019 Aleksej Komarov
- * SPDX-License-Identifier: GPL-2.0-or-later
- */
```

Comment:
are you relicensing these?

Reply:
i don't really care about licensing these specific files, so yes.